worker/eval.ml

```diff
@@ -141,3 +141,8 @@ let execute ~id ~line_number ~output code_text =
 let () =
   Ast_mapper.register_function :=
     fun _ f -> ppx_rewriters := f :: !ppx_rewriters
+
+module Force_aliases = struct
+  include Stdlib.Either
+  include Stdlib.Bigarray
+end
```

Comment:
The other issue was that this two modules are not registered in the jsoo runtime (causing `undefined` errors when loading libraries that depend on them), even though it looks like their code is exported?... This hack forces their inclusion; but then I'm sure there are other module aliases that are missing? I tried playing around with `--linkall` but it didn't help.